client/js/_author/components/bank_navigation/bank_list.jsx

```diff
@@ -19,6 +19,8 @@ export default function bankList(props) {
               {
                 _.map(props.banks, bank => (
                   <ListItem
+                    baseEmbedUrl={props.baseEmbedUrl}
+                    showEmbedCode={(assessId, bankId) => { props.showEmbedCode(assessId, bankId); }}
```

Comment:
you don't actually need a lambda here - you can just pass props.showEmbedCode. It's already correctly bound to the parent by the arrow function in the parent component.